vmm/src/device_manager.rs

```diff
@@ -2555,6 +2555,7 @@ impl DeviceManager {
                 virtio_devices::Balloon::new(
                     id.clone(),
                     balloon_config.size,
+                    balloon_config.deflate_on_oom,
```

Comment:
Please add some integration testing to validate this feature.

Reply:
> Please add some integration testing to validate this feature (if possible).

Ok, sorry for the late reply. We will prepare a corresponding integration testing ASAP. :)